style.css

```diff
@@ -2854,3 +2848,172 @@ h4:hover .learn-more, h5:hover .learn-more, h6:hover .learn-more {
+    overflow: hidden;
+    line-height: 0;
+    margin: 1em;
+}
+
+.fancybox img {
+    display: inline-block;
+}
+
+.fancybox .overlay {
+    display: none;
+    height: 100%;
+    position: absolute;
+    text-decoration: none;
+    width: 100%;
+    z-index: 100;
+    background-color: rgba(255,103,96, 0.92);
+    padding: 20px;
+    line-height: 24px;
+}
+.fancybox .overlay h3 {
+    color: #FFF;
+    font-weight: 300;
+    font-size: 22px;
+    text-transform: uppercase;
+}
+.fancybox .overlay span.meta {
+    color: #FFF;
+    font-weight: 300;
+    font-style: italic;
+}
+#fancybox-thumbs li {
+    margin: initial;
+}
+
+/* Fancybox Style 2 */
+.fancybox .link {
+    position: absolute;
+    bottom: 10px;
+    right: 10px;
+    background: #ebebeb;
+    -webkit-border-radius: 3px;
+    border-radius: 3px;
+    color: #4a4a4a;
+    width: 30px;
+    height: 30px;
+    text-align: center;
+    font-size: 15px;
+    padding-top: 3px;
+    -moz-opacity: 0;
+    filter: alpha(opacity=0);
+    opacity: 0;
+    -webkit-transition: all 200ms ease-in;
+    -o-transition: all 200ms ease-in;
+    -moz-transition: all 200ms ease-in;
+    line-height: 20px;
+}
+.fancybox:hover .link {
+    -moz-opacity: 1;
+    filter: alpha(opacity=100);
+    opacity: 1;
+}
+.fancybox .link:hover {
+    background-color: #ff6760;
+    color: #FFF;
+}
+.fancybox .link:before {
+    display: block;
+    content: "\f50d";
+    font-family: 'fontello';
+    text-align: center;
+}
+.fancybox .link.chain:before { /* ??? */
+    content: "\f517";
+}
+
+/* Carousel Box gallery */
+.gallery-carousel-box .category-wrapper {
+    padding: 1em;
+    border: 1px solid #ccc;
+    border-radius: 3px;
+    background: #f5f5f5;
+}
+.gallery-carousel-box .category-wrapper .category-info {
+    width: 100%;
+    height: auto;
+    float: none;
+    padding: 0.5em;
+    margin-bottom: 10px;
+    background: none;
+}
+.gallery-carousel-box .category-wrapper .category-info h3 {
+    margin: 0 1em 0 0;
+}
+.gallery-carousel-box .category-wrapper .category-info h3,
+.gallery-carousel-box .category-wrapper .category-info p:first-of-type {
+    display: inline-block;
+}
+.gallery-carousel-box .category-wrapper .carousel-wrapper {
+    width: 100%;
+    float: none;
+}
+.gallery-carousel-box .category-wrapper .carousel-wrapper .touch-carousel .item-block img {
+    height: initial;
+}
+
+/* Portfolio */
+.item-info li a {
+    color: inherit;
+}
+.item-info li a:hover {
+    color: #2EAE9B;
+}
+
+.items {
+    padding-left: 0;
+}
+
+.portfolio-detail-view code,
+.portfolio-detail-view pre {
+    background: #444;
+    border-color: #888;
+}
+
+/* Plugins */
+.items li.defaultThumbnail a .overlay {
+    display: block !important;
+    opacity: 1 !important;
+    background-color: transparent;
+    transition: background-color 200ms linear 0s;
+}
+
+.items li.defaultThumbnail a .overlay:hover {
+    background-color: rgba(46,174,155, 0.92);
+    transition: background-color 300ms linear 0s;
```

Comment:
You don't need a `transition` on both `.overlay` and `.overlay:hover`, you can remove this one.